commit-status-tracker/pkg/controller/pipelinerun/pipelinerun_controller_test.go

```diff
@@ -354,7 +354,7 @@ func makeReconciler(pr *pipelinev1.PipelineRun, objs ...runtime.Object) (*Reconc
 	s.AddKnownTypes(pipelinev1.SchemeGroupVersion, pr)
 	cl := fake.NewFakeClient(objs...)
 	client, data := fakescm.NewDefault()
-	fakeClientFactory := func(s string) *scm.Client {
+	fakeClientFactory := func(s, repoType string) *scm.Client {
```

Comment:
One thing you could do here, is assert that `repoType` matches what you expect, and fail the test if not?